Dockerfile

```diff
@@ -10,6 +10,7 @@ WORKDIR /nodeApp
 ADD . /nodeApp
 RUN ./make_version.sh
 
+# why is this not "npm install --production"? I'd like to exchange it with "npm prune --production"
```

Comment:
Sounds well thought and I can't see why it shouldn't work.

@anyone else: Any other reasons for keeping dev deps as part of the production environment?